demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -666,13 +666,32 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
     }
   }
 
+  async createLogStream(configuration: MeetingSessionConfiguration): Promise<void> {
+    const body = JSON.stringify({
+      meetingId: configuration.meetingId,
+      attendeeId: configuration.credentials.attendeeId,
+    });
+    try {
+      const response = await fetch(`${DemoMeetingApp.BASE_URL}create_log_stream`, {
+        method: 'POST',
+        body
+      });
+      if (response.status === 200) {
+        console.log('Log stream created');
```

Comment:
Do we need to handle for any other statuses here? Like throw error when access related error comes in. Not sure about the working here, but if we get a security token exception for the fetch call is it catched in `response.status` or in catch block.

Reply:
createLogStream is called as soon as the meeting session is initialized. So token exception for the fetch is unlikely to occur.
Moreover the handler has logic to create a stream if it does not exist.